services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -464,6 +445,66 @@ private UserConsentModel createConsent(ClientModel client, ConsentRepresentation
         return consent;
     }
 
+    @Path("/applications")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @NoCache
+    public Response applications() {
+        checkAccountApiEnabled();
+        auth.requireOneOf(AccountRoles.VIEW_APPLICATIONS, AccountRoles.MANAGE_APPLICATIONS);
```

Comment:
@douglaspalmer I believe `MANAGE_APPLICATIONS` should be used only in the removal endpoint, but for listing `VIEW_APPLICATIONS` should be enough. Does it make sense?

Maybe worth to ask on the dev mailing list and engage with others to see what they think.

Reply:
@douglaspalmer I plugged react with the Applications to the endpoint. The correct combo of permissions seems to be:
```
auth.requireOneOf(AccountRoles.VIEW_APPLICATIONS, AccountRoles.VIEW_PROFILE);
```
Otherwise, I get HTTP 403.